src/pages/docs/chat/getting-started/react-ui-components.mdx

```diff
@@ -0,0 +1,460 @@
+---
+title: "Getting started: Chat UI Components with React"
+meta_description: "Step-by-step quick-start for ably-chat-react-ui-components using React and Vite."
+meta_keywords: "Ably, realtime, quickstart, Chat UI, React, Vite"
+---
+
+This guide shows you how to add Ably Chat React UI Components to a brand-new React app built with Vite.
+
+You will:
+
+* Create an Ably realtime client and Chat client
+* Install the ready-made UI component package
+* Build up a complete chat interface component by component
+* Customise look and feel with your own styles
+* Add your own chat settings and avatars
+
+## Prerequisites <a id="prerequisites"/>
+
+1. Sign up for an Ably account.
+
+2. Create an app and copy its API key (the root key is fine while you experiment).
+
+3. Install the Ably CLI:
+
+<Code>
+  ```shell
+  npm install -g @ably/cli
+  ```
+</Code>
+
+4. Log in and set the default app/key:
+
+<Code>
+  ```shell
+  ably login
+  ably apps switch
+  ably auth keys switch
+  ```
+</Code>
+
+<Aside data-type="note">
+  The snippets below use a demo API key. Replace it with your own before going live.
+</Aside>
+
+### Create a new project <a id="prerequisites-project"/>
+
+1. Scaffold a **React + TypeScript** project with Vite:
+
+<Code>
+  ```shell
+  npm create vite@latest my-chat-ui-app -- --template react-ts
+  cd my-chat-ui-app
+  ```
+</Code>
+
+2. (Optional) Add Tailwind CSS if you want utility classes elsewhere in your app. The UI kit itself ships pre-compiled CSS, so Tailwind isn't required:
+
+<Code>
+  ```shell
+  npm install -D tailwindcss postcss autoprefixer
+  npx tailwindcss init -p
+  ```
+</Code>
+
+Follow the Tailwind Vite guide to configure the generated files.
+
+3. Install the Chat UI kit and its peers:
+
+<Code>
+  ```shell
+  npm install ably-chat-react-ui-components
```

Comment:
Package name

Reply:
Also need to install `@ably`.

I would do this as one command:

`npm install ably @ably/chat @ably/chat-react-ui-components`